pkg/app/handlers/users/handler.go

```diff
@@ -251,3 +234,35 @@ func (u UsersResourceHandler) setUserMappings(ctx context.Context, userID string
 	}
 	return nil
 }
+
+func (u UsersResourceHandler) getIdentityRelation(userID, identity string) *dsc.Relation {
+	rel := strings.Split(u.cfg.SCIM.IdentityRelation, "#")
+	if len(rel) == 2 {
+		switch rel[0] {
+		case "user":
```

Comment:
Here too there needs to be input validation. `rel[0]` doesn't have to be "user" or "identity" but it does need to be either `UserObjectType` or `IdentityObjectType`.